lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "lerna": "2.4.0",
+  "lerna": "3.22.1",
```

Comment:
!!!!!!!

Reply:
There's no `--cd-version` in 3.x so we need to rephrase the command execution for this to work, really